lighthouse-core/scripts/chromium-web-tests/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
🚲 🏠 this name, `test-locally.sh` since we never intend to run this in CI?

Reply:
I don't like it. The test process is 100% the same, but broken up for Actions because more granular steps makes debugging easier. Also needed some closer control over the env variables.

We don't call our local smoke tests "local-smoke" or w.e.

Reply:
> The test process is 100% the same, but broken up for Actions because more granular steps makes debugging easier. Also needed some closer control over the env variables.

I appreciate all that which is why my comment wasn't "Let's run in this in CI instead of duplicating the script in yaml" :)

> We don't call our local smoke tests "local-smoke" or w.e.

I don't understand how that compares. We literally run `yarn smoke <test name>` both locally and in CI so there's no need to differentiate because they're the same. This is a test script that is only ever intended to work locally, and explicitly won't be run in CI.